lighthouse-extension/app/src/report-loader.js

```diff
@@ -42,16 +42,34 @@ class ReportLoader {
 
   write(results) {
     const reportGenerator = new ReportGenerator();
-    const html = reportGenerator.generateHTML(results);
-
-    this.stopSpinner();
-    document.documentElement.innerHTML = html;
+    try {
+      const html = reportGenerator.generateHTML(results);
+      this.stopSpinner();
+      document.documentElement.innerHTML = html;
+    } catch (err) {
+      this.stopSpinner();
+      this.displayException(err, results);
+    }
 
     // Find and replace all the scripts that have been injected so that they get evaluated.
     const scripts = document.querySelectorAll('script');
     scripts.forEach(this.replaceScript);
   }
 
+  displayException(err, results) {
+    const html = `
+      <div class="report-error">
+        <h1 class="error-message">⚠️ Error: <span></span></h1>
+        <p class="error-stack"><span></span></p>
+        <big>➡ <a target="_blank" href="httbigs://github.com/GoogleChrome/lighthouse/issues">Please report this bug</a></big>
```

Comment:
🎶  they like it when you call be big https 🎵 

(httbigs) is a pretty cool scheme but I do not think it is supported everywhere yet :(